t/14-isotovideo.t

```diff
@@ -105,6 +106,35 @@ subtest 'isotovideo with git refspec specified' => sub {
             opts => "casedir=$data_dir/tests test_git_refspec=deadbeef _exit_after_schedule=1") } qr/Checking.*local.*deadbeef/, 'refspec in local git repository would be checked out';
 };
 
+subtest 'isotovideo with wheels' => sub {
+    chdir($pool_dir);
+    unlink('vars.json') if -e 'vars.json';
+    my $specfile = path("$data_dir/tests/wheels.yaml");
+
+    my $system_mock = Test::MockModule->new('IPC::System::Simple');
+    $system_mock->redefine(capture => sub { return; });
```

Comment:
Apparently this doesn't work. `git` still seems to be called. I don't understand it...

Reply:
Marius suggested it doesn't work because it's not a qualified function on a blessed object (c.f. [Mojo::File / slurp](https://github.com/os-autoinst/os-autoinst/blob/master/t/10-virtio_terminal.t#L74))

Reply:
I've decided to drop the tests that would require mocking and tackle that in *pr coming up, please refresh later* instead